vagrant_repo/data_bags/cerner_splunk/license-vagrant.json

```diff
@@ -0,0 +1,14 @@
+{
+  "id": "license-vagrant",
+  "auto_generated_pool_size": "100KiB", 
+  "pools":{
+    "pool1": {
+      "max_size": "200KiB",
+      "GUIDS": "aae84706-a70d-4134-951d-93a6541011fa, 9d8f6bb8-49b2-463e-afdd-aedf267526e4"
```

Comment:
Nit: An array is a natively supported structure in JSON... we probably should have the list of GUIDs be an actual list and translate for the user, instead of relying on the user to put the proper separators into the list for Splunk. Also this should make it easier to generate a config file based on a database, chef search, something else.